packages/nuxt/src/runtime/plugins/sentry.server.ts

```diff
@@ -0,0 +1,23 @@
+import { captureException } from '@sentry/node';
+import { H3Error } from 'h3';
+import { defineNitroPlugin } from 'nitropack/runtime';
+
+export default defineNitroPlugin(nitroApp => {
+  nitroApp.hooks.hook('error', (error, context) => {
+    // Do not handle 404 and 422
+    if (error instanceof H3Error) {
+      if (error.statusCode === 404 || error.statusCode === 422) {
+        return;
+      }
+    }
+
+    if (context) {
+      captureException(error, {
+        captureContext: { extra: { nuxt: context } },
```

Comment:
Ideally we avoid setting `extra`, it is soft deprecated.

What is contained in this context? Can we create a `nuxt` contexts instead? Or more intelligently parse `contexts`?

Reply:
If this is true, sorry @s1gr1d for misleading.

Reply:
This is included:
```js
{
  event: '[GET] /api/test-param/1234',
  tags: ['request']
}
```

So this could be changed to this?
```suggestion
        captureContext: { context: { nuxt: context } },
```